Anagram/AnagramList.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+
+namespace Anagram
+{
+    public class AnagramList
+    {
+        public List<string> SelectAnagrams(string word, List<string> candidates)
+        {
+            //Insert the correct implementation here
+            List<string> anagramList = new List<string> { };
+            foreach (string str in candidates)
+            {
+                CheckAndAddAnagramsToList(word, str, anagramList);
+            }
+            return anagramList;
+        }
+        public List<string> CheckAndAddAnagramsToList(string word, string str, List<string> anagramList)
+        {
```

Comment:
A good split of complexity between functions and nice naming 👍 
Suggestion: If you return a value from a function, use it in the caller. Otherwise, make the return type as void.